contracts/RewardPool.vy

```diff
@@ -140,39 +140,6 @@ def _find_timestamp_epoch(_timestamp: uint256) -> uint256:
     return _min
 
 
-@view
-@internal
-def _find_timestamp_user_epoch(user: address, _timestamp: uint256, max_user_epoch: uint256) -> uint256:
-    _min: uint256 = 0
-    _max: uint256 = max_user_epoch
-    for i in range(128):
-        if _min >= _max:
-            break
-        _mid: uint256 = (_min + _max + 2) / 2
-        pt: Point = VEYFI.point_history(user, _mid)
-        if pt.ts <= _timestamp:
-            _min = _mid
-        else:
-            _max = _mid - 1
-    return _min
-
-
-@view
-@external
-def ve_for_at(_user: address, _timestamp: uint256) -> uint256:
```

Comment:
with the new method, is there a replacement way to get this info? seems useful for debugging issues and data off chain

Reply:
ok just to be clear this method is now replaced with balanceOF?

Reply:
Yes